src/components/manage/Form/__snapshots__/InlineForm.test.jsx.snap

```diff
@@ -66,7 +66,7 @@ exports[`Form renders a form component with defaults in the schema - Checkboxes
            - 
           No description
            -
-          No value
+          false
```

Comment:
This is a bug fix. The test was faulty. It explicitely says "Form renders a form component with defaults in the schema - Checkboxes", passes "false" as value to the checkbox, so this is correct.